src/components/app-progress-bar-line/AppProgressBarLine.tsx

```diff
@@ -7,23 +7,48 @@ import { styles } from '~/components/app-progress-bar-line/AppProgressBarLine.st
 import { LinearProgress } from '@mui/material'
 import { UserRoleEnum } from '~/types'
 
+import Image from '~/assets/img/cooperation-details/clock.svg'
+
 interface AppProgressBarLineProps {
   value: number
   userRole: UserRoleEnum | ''
+  isCooperationActivities?: boolean
 }
 
 const AppProgressBarLine: FC<AppProgressBarLineProps> = ({
   value,
-  userRole
+  userRole,
+  isCooperationActivities = false
 }) => {
   const { isMobile } = useBreakpoints()
   const labelsValue =
     userRole === UserRoleEnum.Student
       ? [0, 25, 50, 75, 100]
       : [0, 20, 40, 60, 80, 100]
-
-  const labelsWithPercent = isMobile ? (
-    <Typography color={'primary.500'} variant='subtitle2'>
+  const labelsWithPercentForCooperation = (
+    <Box width={'100%'}>
+      <Typography color={'primary.500'} fontSize={'12px'} variant='subtitle1'>
+        Your progress
+      </Typography>
+      <Box sx={styles.wrapperTypographyProgressCoop}>
+        <Typography color={'#2B6E6E'} fontSize={'20px'} variant='h5'>
+          {`${value}% completed`}
+        </Typography>
+        <Box display={'flex'}>
+          <Box component='img' marginRight={'8px'} src={Image} />
+          <Typography
+            color={'primary.500'}
+            fontSize={'12px'}
+            variant='subtitle1'
+          >
+            {`${100 - value}% to complete`}
+          </Typography>
+        </Box>
+      </Box>
+    </Box>
+  )
+  const labelsWithPercentForProfile = isMobile ? (
+    <Typography color='primary.500' variant='subtitle2'>
```

Comment:
1. `"Width"`, `"display"`, `"mb"` and so on move to style files
2. For typography we have specific predefined values. Example: `typography: TypographyVariantEnum.Body1`
3. Do not use hardcoded color codes. Use from palette